packages/components/src/Typography/css/Emphasis.css

```diff
@@ -3,16 +3,14 @@
 }
 
 .highlight {
-  margin-left: calc(-1 * var(--space-smaller));
-  padding: 0 var(--space-smaller);
+  display: inline;
```

Comment:
The linter gets upset that I'm using display: inline and margin on the same element